clustering.ipynb

```diff
@@ -64,6 +64,8 @@
     "import pandas as pd\n",
```

Comment:
<p>No correlation resulted!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Reema-Khaseeb/DM-Assignment1-NYC-transportation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
the column will be dropped anyways